tests/src/performance/Scenario/JitBench/JitBenchHarness.cs

```diff
@@ -209,7 +209,6 @@ private static void DirectoryCopy(string sourceDir, string destDir)
                 FileName = "powershell.exe",
                 Arguments = $"-Command \".\\RunBenchmark.ps1 " +
                             $"-SetupOnly " +
-                            $"-PrecompiledViews " +
                             $"-Architecture {s_targetArchitecture} " +
```

Comment:
How will this change impact performance?

Reply:
WIth and without precompiled views is effectively two different benchmarks. The current benchmark appears not to use precompiled views so I made this change here to keep the behavior the same.

Precompiled views themselves is an ASP.Net feature that runs csc on the view code at build time rather than at first execution time. It makes the time we measure for first request much smaller because the app doesn't have to load and JIT Roslyn as part of completing the first request.

Reply:
Would it make sense to have an option to profile both?

Reply:
yeah it would, but since that would involve a larger set of changes I didn't want to bite that off just yet.

Reply:
Makes sense :)